src/visualstates/core/parameter.py

```diff
@@ -57,11 +57,14 @@ def parseElement(self, element):
                 self.name = str(value)
             elif name == 'type':
                 self.type = str(value)
-
-        self.value = str(element.getElementsByTagName('value')[0].childNodes[0].nodeValue)
-        self.desc = str(element.getElementsByTagName('description')[0].childNodes[0].nodeValue)
-        print(self.name)
-        print(self.value)
+        try:
+            self.value = str(element.getElementsByTagName('value')[0].childNodes[0].nodeValue)
+        except IndexError:
+            self.desc = ''
```

Comment:
Is this supposed to be `self.value = ''` ?

Reply:
Yes thanks for pointing it out